app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertyInformationCustomView.java

```diff
@@ -43,13 +44,21 @@ private void initView(TypedArray attributes) {
     }
 
 
-    public static void setValueText( View view,String text) {
+    public static void setValueText(@NonNull View view, String text) {
         EditText editText = view.findViewById(R.id.add_property_information_custom_view_value);
         editText.setText(text);
     }
 
-    public static String getValueForView(View view){
+
+    @NonNull
```

Comment:
Method does not need to be static
editText should be global to the CustomView. And you won't need to pass the view as parameter

Reply:
done